ui/src/theme.ts

```diff
@@ -6,7 +6,8 @@ const PRIMARY_LIGHTER = '#DDE0FF';
 const PRIMARY_LIGHTEST = '#EEEFFF';
 
 const SECONDARY_MAIN = '#9E9E9E';
-const SECONDARY_LIGHT = '##00000099';
+const SECONDARY_LIGHTER = '#999999';
+const SECONDARY_LIGHT = '#00000099';
```

Comment:
what is this code ya? Color codes should only have 6 digits.
Also don't forget to add the second lighter in the `customTheme` if ur gonna use it.